src/App.jsx

```diff
@@ -1,33 +1,64 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useState } from "react"
+import "./styles.css"
+import NewToDoForm from "./NewToDoForm"
```

Comment:
as you are exporting `NewToDoForm` not as a default you need to change here the way you import
```
import { NewToDoForm } from "./NewToDoForm"
```